queries.sql

```diff
@@ -23,3 +23,32 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg).
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'Digmon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'Pokemon' WHERE species IS NULL;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT sp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO sp1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT MAX(escape_attempts) FROM animals;
+SELECT MIN(weight_kg), MAX(weight_kg) FROM animals;
```

Comment:
- [ ] Kindly make `MAX` and `MIN` in one query.

Reply:
I would make it two queries if there were `,` before `and`.

![Screen Shot 2022-10-19 at 17 06 35](https://user-images.githubusercontent.com/69011963/196686125-b7cbf930-91f9-40ba-ad7d-38857f059fe5.png)

Reply:
It's one requirement and not two. Also, I would make it two queries if there were `,` before `and`.

![Screen Shot 2022-10-19 at 17 06 35](https://user-images.githubusercontent.com/69011963/196686967-514eb380-9af6-4cb1-9097-6adfdd91abc7.png)